### DISCUSSION TITLE
Changes on GitHub aren't showing up on the website

### DISCUSSION THREAD

**Discussion by mlbeanmuseum** [community/community#64094] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

We use GitHub to publish interactive programs that we use at our business to educate and interact with patrons. We've been using GitHub for years without any issues until recently. We are putting together a new exhibit so I've created new programs for it using the same software program as usual (Articulate Storyline 360) and uploaded them to GitHub using the same process as usual. On the GitHub website, they show up in the ReadMe main page as they should, but when I go to the actual website they aren't there and none of the changes have gone through. Does anyone know why this is happening or how I can fix this?

**Comment by christophercouch** [comment]
Hi @mlbeanmuseum,

It sounds frustrating to face such an issue, especially when you've been using GitHub consistently without problems. Here are a few troubleshooting steps and suggestions you might consider:

GitHub Pages Cache: Sometimes, GitHub Pages (assuming you're using this to host) might serve a cached version of your website. You can try accessing the website in an incognito/private browsing mode or try clearing your browser's cache to see if the latest changes reflect.

Build Errors: If you're using Jekyll (or another static site generator) with GitHub Pages, there might be build errors preventing the updated content from being displayed. Check the Settings section of your repository and navigate to the GitHub Pages section to see if there are any build warnings or errors.

DNS Issues: If you're using a custom domain, it might be worth checking if there's any DNS misconfiguration. It's possible for the domain to point to an older version of the site if not properly configured.

Commit History: Double-check the commit history of your repository to ensure that the changes were indeed pushed to the correct branch (typically gh-pages or main/master depending on your setup).

Page Routing: If you're using any form of routing or directory structure, ensure that the paths are correctly configured. Sometimes, a small misconfiguration can prevent certain pages or assets from displaying.

Repository Settings: Ensure that you're pushing to the correct branch that's set up for GitHub Pages deployment. If you've recently changed the branch (e.g., from master to main), you might need to update the GitHub Pages settings to reflect that.

Manual Deployment: As a temporary measure, you might consider deploying manually using the gh-pages npm package or another tool to ensure that the website's most recent version gets pushed.

Contact Support: If none of the above solutions work, you might consider reaching out to GitHub support. They can often provide insights specific to your repository's configuration and deployment process.

I hope one of these solutions helps resolve your issue.
